src/main/java/org/opentripplanner/routing/algorithm/raptor/transit/Transfer.java

```diff
@@ -11,20 +13,33 @@
 
     private final List<Coordinate> coordinates;
 
+    private final List<Edge> edges;
+
     public Transfer(int toStop, int distanceMeters, List<Coordinate> coordinates) {
         this.toStop = toStop;
         this.distanceMeters = distanceMeters;
         this.coordinates = coordinates;
+        this.edges = Collections.emptyList();
+    }
+
+    public Transfer(int toStop, int distanceMeters, List<Coordinate> coordinates, List<Edge> edges) {
+        this.toStop = toStop;
+        this.distanceMeters = distanceMeters;
+        this.coordinates = coordinates;
+        this.edges = edges;
     }
 
     public List<Coordinate> getCoordinates() {
         return coordinates;
     }
 
-    //TODO getToStop
-    public int stop() { return toStop; }
+    public int getToStop() { return toStop; }
```

Comment:
The reason I left this as "stop" instead of "toStop", is that the Transfer class is also used for both access and egress legs. In the case of egress leg it would be "from stop".